### PR TITLE
starting jupyter + more docs + pkg-ing!

### LINKED ISSUE #35: improving webpage
- [ ] select style, etc.
- [ ] add content
- [ ] automate pop.

### LINKED ISSUE #36: start jupyterbook
- [ ] demos of each module, class, etc. 
- [ ] need to set up a pypi/conda pkg soon? otherwise a little cumbersome

### LINKED ISSUE #38: split github action workflows
it may make more sense to split everything:

1. pylint
2. pytest
3. flake8
4. additional testing
5. package building/ci stuff

### LINKED ISSUE #39: ignore jupyterbook in pylint and flake8

### PR DESCRIPTION
**Pull request by ngam**
Fixes #36

Fixes #35 (maybe!)

Fixes #39 

Fixes #38

### CODE REVIEW

**Comment by ngam** [comment]
@Gorkowski @wkchuang looks intimidating for review --- sorry. I would advise you to only look at changes in `aerosol_dynamics/particle.py` as the rest is simply to do with drafted housekeeping stuff (webpage, contributing file, codeowners file, etc.) that will be more properly vetted later.

**Comment by ngam** [comment]
@wkchuang let me know if you want me to 1) wait until your PR goes through, 2) merge your PR into this PR, or 3) take care of your PR after merging this PR. There shouldn't be much conflict.

File: aerosol_dynamics/physical_parameters.py
Comment (Gorkowski) [inline aerosol_dynamics/physical_parameters.py:14]: good catch

Comment (ngam) [reply aerosol_dynamics/physical_parameters.py:14]: The beauty of having unit checks everywhere!

Comment (wkchuang) [reply aerosol_dynamics/physical_parameters.py:14]: I should have pushed out a hotfix for that, sorry! I caught it while working on the Environment class (yes, units checks!) but thought I'd be able to get it out faster.

Comment (ngam) [reply aerosol_dynamics/physical_parameters.py:14]: @wkchuang that's why it is good to do smaller commits on the fly like I do --- your work won't be hidden from us. Our PRs ad commits don't have to be perfect, as long as we are getting stuff done. Public mistakes are fine --- after all, that's the main way I've been learning all of this!

File: aerosol_dynamics/particle.py
Comment (wkchuang) [inline aerosol_dynamics/particle.py:230]: Should we add a units decorator here?

Comment (ngam) [reply aerosol_dynamics/particle.py:230]: Will do, thanks!

**Review by wkchuang — APPROVED** [review]

**Comment by ngam** [comment]
🚀

**Pull request merged**
